src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2004,7 +2004,7 @@ namespace winrt::TerminalApp::implementation
         auto profiles = _settings->GetProfiles();
         for (auto& profile : profiles)
         {
-            const GUID profileGuid = profile.GetGuid();
+            const GUID profileGuid = GUID(profile.Guid());
```

Comment:
do not use `()` ctor syntax.

Reply:
in fact, this should work since the type is named on the left.

```suggestion
            const GUID profileGuid{ profile.Guid() };
```